### ISSUE TITLE
[Priest] casting pushback very unbalanced when fight two enemys or more

### ISSUE THREAD

**Issue by Bootsbutera** [The-Alpha-Project/alpha-core#569]
Regardless of what casting spell used, the casting pushback is not balanced when fighting two or more enemys. 
Tested with different enemys like human enemys and beast enemys, issue is the same.
Even with just two enemys its almost impossable to get a spell casting complete and even worse the more enemys it is.

**Comment by GizzleBizzle** [comment]
@Bootsbutera 
It is not a bug. It should be even worse, but crushing blows aren't implemented, yet. (Taken from 0.5.3 patch notes)
![obraz](https://user-images.githubusercontent.com/111394734/186502781-5e18397f-c227-4f9b-a995-7d621ab37087.png)

It is nice to always provide source of information of the thing You want to change. WoW mechanics have changed a lot over the years and they can differ vastly even one month later, so that's why it's important to have a proof of what we claim so everyone's sure it is supposed to be this way.

**Comment by Fluglow** [comment]
This may have been an oversight on my part when implementing cast pushback. Is there any evidence that pushback should be uncapped like it is now?

**Comment by GizzleBizzle** [comment]
![Stitches (Abomination)](https://user-images.githubusercontent.com/111394734/194937995-7b7eaba6-597b-4234-8f3b-402a4e7f6d30.jpg)

I think pushback estimates are in the realm of possible correctness right now.
Pulled two moonstalkers (like the guy says in the chat) and quite frankly a caster would have problems casting anything. I finished one cast after solid 20 seconds of casting because misses and blocks alligned. Definitely not something a similar-lvled character would survive.

The thing I'm not sure about, however is the negative cast time I experience in the video I captured:
https://user-images.githubusercontent.com/111394734/194940147-79e84cbd-018c-4c55-9382-06d72584872b.mov
I wonder:
- if it is purely a visual bug or if getting hit can actually push the casting into negative values (from 0 seconds to 3 into -1 second to 3).
- If it's not a visual bug, should it work like that.
